sections/syntax.include

```diff
@@ -4134,31 +4077,7 @@
     :: If the DOCTYPE token's name is not a [=case-sensitive=] match for the string "`html`", or the
         token's public identifier is not missing, or the token's system identifier is neither
         missing nor a [=case-sensitive=] match for the string
-        "<a scheme><code>about:legacy-compat</code></a>", and none of the sets of conditions in the
-        following list are matched, then there is a [=parse error=].
-
-        * The DOCTYPE token's name is a [=case-sensitive=] match for the string "`html`", the
-            token's public identifier is the [=case-sensitive=] string
-            "`-//W3C//DTD&nbsp;HTML&nbsp;4.0//EN`", and the token's system identifier is either
-            missing or the [=case-sensitive=] string "`http://www.w3.org/TR/REC-html40/strict.dtd`".
-        * The DOCTYPE token's name is a [=case-sensitive=] match for the string "`html`", the
-            token's public identifier is the [=case-sensitive=] string
-            "`-//W3C//DTD&nbsp;HTML&nbsp;4.01//EN`", and the token's system identifier is either
-            missing or the [=case-sensitive=] string "`http://www.w3.org/TR/html4/strict.dtd`".
-        * The DOCTYPE token's name is a [=case-sensitive=] match for the string "`html`", the
-            token's public identifier is the [=case-sensitive=] string
-            "`-//W3C//DTD&nbsp;XHTML&nbsp;1.0&nbsp;Strict//EN`", and the token's system identifier
-            is the [=case-sensitive=] string "`http://www.w3.org/TR/xhtml1/DTD/xhtml1-strict.dtd`".
-        * The DOCTYPE token's name is a [=case-sensitive=] match for the string "`html`", the
-            token's public identifier is the [=case-sensitive=] string
-            "`-//W3C//DTD&nbsp;XHTML&nbsp;1.1//EN`", and the token's system identifier is the
-            [=case-sensitive=] string "`http://www.w3.org/TR/xhtml11/DTD/xhtml11.dtd`".
-
-        Conformance checkers may, based on the values (including presence or lack thereof) of the
-        DOCTYPE token's name, public identifier, or system identifier, switch to a conformance
-        checking mode for another language (e.g., based on the DOCTYPE token a conformance checker
-        could recognize that the document is an HTML 4.01-era document, and defer to an HTML 4.01
-        conformance checker.)
```

Comment:
Why remove the statement about conformance checkers? It is a no-op, but it seems reasonable…